docs/source/development/automated_testing.md

```diff
@@ -96,6 +96,17 @@ Tests should be named as descriptively as possible, especially if you are workin
 
 You can read more about the [basics of using `pytest` on the getting started page](https://docs.pytest.org/en/7.1.x/getting-started.html). For help writing your own tests and using all of the features of `pytest`, see the [project documentation](https://docs.pytest.org/).
 
+### Specify `pythonpath` in `pyproject.toml`
+
+Specify your source directory in `pyproject.toml` for pytest to properly discover your Kedro project. This is necessary due to the nested directory structure.
+
+```toml
+[tool.pytest.ini_options]
+pythonpath = [
+  "src"
+]
+```
+
```

Comment:
I am not particular familiar with this config, but looks like kedro starter don't have this config?

Reply:
Opened a PR to add it, it resolves the issue of "Module spaceflights not found" when importing from the project

Reply:
This works, but I think the better solution is to tell users to install their own code:

```
pip install -e .
```

See also https://github.com/kedro-org/kedro/issues/3418

Reply:
Changed in [9339ad1](https://github.com/kedro-org/kedro/pull/3782/commits/9339ad14acc54c88f28e290fb85795e273322701)